data/templates/Detector_ArcAppliance.csx

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+        let startTime = datetime({cxt.StartTime});
+        let endTime = datetime({cxt.EndTime});
+        cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
```

Comment:
Please change this line 
FROM cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
To YOUR_TABLE_NAME

This is due to a recent change in the platform. The documentation has now been updated.

Reply:
Fixed in latest commit